cl/users/email_handlers.py

```diff
@@ -143,17 +145,16 @@ def handle_soft_bounce(
                         # Check if backoff event has reached
                         # max number of retries, if so ban email address
                         # Only ban email address if not previously banned
+                        max_retry_subtype = "MaxRetryReached"
```

Comment:
Why not remove this and just do `EMAIL_NOTIFICATIONS.MAX_RETRIES_REACHED`?

Reply:
removed and use the previous solution: `EMAIL_NOTIFICATIONS.INVERTED["MaxRetryReached"]`